NEWS.md

```diff
@@ -26,6 +27,113 @@ The format is based on [Keep a Changelog]
+        <map button="Stylus2" to="Middle" />
+      </tablet>
+
+- Add tearing support. #1390. Written-by: @Ph42oN @ahesford
+- Add configuration support for mouse buttons `Side`, `Extra`, `Forward`,
+  `Back` and `Task`. Written-by: @jp7677
+- config: allow `<libinput><device>` without category attribute to define a
+  `default` profile because it is more user-friendly and intuitive.
+- Add a configuration option enable adaptive sync only when an application is
+  in fullscreen mode. Written-by: @Ph42oN
+- Add `touchpad` libinput device type to increase configuration flexibility,
+  for example allowing `naturalScroll` on touchpads, but not on regular pointer
+  devices such as mice. Written-by: @jmbaur
+- Add actions:
+    - `AutoPlace` (by @ahesford)
+    - `MoveToOutput`, `FitToOutput` (by @jp7677)
+    - `Shade`, `Unshade`, `ToggleShade` (by @ahesford @Consolatis)
+- Add config option `<placement><policy>` with supported values `center`,
+  `under-cursor` and `automatic`. The latter minimizes overlap with other
+  windows already on screen and is similar to Openbox's smart window placement.
+  Written-by: @ahesford #1312
+
+      <placement>
+        <policy>center|automatic|cursor</policy>
+      </placement>
+
+### Fixed
+
+- Fix output configuration bug causing compositor crash when refresh rate is
+  zero. Issue #1458
+- Fix disappearing cursor bug on view destruction. Issue #1393
+- Use used specified config-file (using -c command line option) on
+  reconfigure.
+- view: assign output on surface creation instead of mapping and notify
+  the client of the preferred output scale when doing so. This fixes an
+  issue with foot: https://codeberg.org/dnkl/foot/issues/1579
+  Written-by: @ahesford
+- Cancel key repeat on vt change to fix crash on VT change on FreeBSD.
+  Issue #1424
+- Fix crash when a minimized fullscreen window closes. Written-by: @bi4k8
+- Execute menu actions after closing menus so that menu entries can issue
+  `wtype` commands to the surface with keyboard-focus. Issue #1366
+- Try to honor original window geometry on layout changes.
+- Fix virtual keyboard bug experienced with `wlrctl keyboard type xyz`. Do not
+  process virtual keyboard keycodes (just the keysyms). Issue #1367
+- Sync xdg-shell client `view->pending` when applying geometry to fix issue
+  caused by applications choosing not respond to pending resize requests either
+  by ignoring them or substituting alternative sizes (for example, when mpv
+  constrains resizes to keep its aspect ratio fixed). Written-by: @ahesford
+    
+### Changed
+
+- The new windowEdgeStrength setting makes windows resist interactive moves and
+  resizes across the edges of other windows. This can be disabled with:
+
+      <resistance>
+        <windowEdgeStrength>0</windowEdgeStrength>
+      </resistance>
+
+- When `$XDG_CONFIG_HOME` is defined, it replaces (rather than augments)
+  `$HOME/.config`. Similarly, `$XDG_CONFIG_DIRS` replaces `/etc/xdg` when
+  defined.
```

Comment:
This was the case before, because the first-processed configuration would terminate the search for configuration files.

Reply:
Whoa... good point. Got too wrapped up in the detail! Have removed that paragraph.